.github/workflows/release-packages.yml

```diff
@@ -135,7 +140,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+
     - uses: cargo-bins/cargo-binstall@main
+    - name: Install binaries required
+      run: cargo binstall -y --force rsign2 rage
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
in this case could add to the install-action below instead

Reply:
Thanks, I've applied ur suggestion